test/box-luatest/meta-temporary_spaces_test.lua

```diff
@@ -0,0 +1,440 @@
+local fio = require 'fio'
+local xlog = require 'xlog'
+local server = require 'luatest.server'
+local cluster = require 'luatest.replica_set'
```

Comment:
We always wrap `require` argument into parentheses. As `require('luatest')`, not `require 'luatest'`.

Reply:
done